Vagrantfile

```diff
@@ -1,11 +1,11 @@
-FLASK_PORT=5000
-POSTGRESQL_PORT=5432
+CLIENT_PORT=5000
+BE_PORT=3000
 
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/bionic64"
   config.vm.provision :shell, path: "setup.sh"
-  config.vm.network :forwarded_port, guest: FLASK_PORT, host: FLASK_PORT
- config.vm.network :forwarded_port, guest: POSTGRESQL_PORT, host: POSTGRESQL_PORT
+  config.vm.network :forwarded_port, guest: CLIENT_PORT, host: CLIENT_PORT
```

Comment:
If I understand you correctly we need instead of forwarding ports, find another way to expose both flask servers to localhost?

Reply:
no, you simply do not need to expect it to be available on localhost. Because Vagrant runs a VM, it feels like a remote machine, and stuff you run on it can be accessed via accessing its IP directly.

There are various ways you could use to make Vagrant print the IP you need to use once you've started it.

For systems that use more complex security mechanisms such as HTTPS, where the actual hostname you type in your browser becomes important, there are tricks one can do with browser plugins like FoxyProxy to make the locally running VM appear as if it was the actual production host. But I don't think you'll need this kind of complexity in this class.

Reply:
Done